src/core/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG BASE_IMAGE=ugbio_base
-FROM python:3.10-bullseye as build
+FROM python:3.11-bullseye as build
```

Comment:
sorry for this unrelated request:
Can you change all those "as" to uppercase "AS" (in all Dockerfiles?)
to prevent the annoying warning on docker build

Reply:
Yes!